__tests__/app.test.js

```diff
@@ -48,6 +48,7 @@ describe("GET /api/articles/:article_id", () => {
           body: expect.any(String),
           created_at: expect.any(String),
           votes: expect.any(Number),
+          comment_count: expect.any(String),
```

Comment:
I'd want you to specifically assert what this comment count should be, just so you're certain the count is doing what you want it to - you know it's article 1 so you should be able to be more specific